tests/cardano_lock_rust/tests/test_cardano_lock.rs

```diff
@@ -120,7 +73,7 @@ fn failed_sign_data() {
     verifier.set_debug_printer(debug_printer);
 
     let verify_result = verifier.verify(MAX_CYCLES);
-    assert!(verify_result.is_err());
+    assert!(verify_result.is_err(), "pass verification");
 }
 
 #[test]
```

Comment:
Add a random test cases. Fill witness lock with random data and loop 100 times: always failed.